test/suites/operations.sh

```diff
@@ -0,0 +1,62 @@
+test_operations_get_all() {
+  assert_get_all_operations() {
+    (
+      set -e
+      echo "==> get all-projects operations normally"
+      result="$(lxc query "/1.0/operations?all-projects=true" | jq '.success[0]' | wc -l)"
+      test "${result}" -eq 1 || false
+
+      echo "==> get all-projects operations recursively"
+      result="$(lxc query "/1.0/operations?recursion=1&all-projects=true" | jq '.success[0]' | grep -c 'status_code')"
+      test "${result}" -eq 1 || false
+    )
+  }
+
+  ensure_import_testimage
+
+  # create container
+  name=x1
+  lxc launch testimage ${name}
+  lxc list ${name} | grep RUNNING
+
+  # get_all_operations
+  lxc query -X POST -d '{\"command\":[\"touch\",\"/root/foo1\"],\"record-output\":false}' /1.0/instances/${name}/exec
+  sleep 1
+  assert_get_all_operations
+  
+  lxc stop "${name}" --force
+  lxc delete "${name}"
```

Comment:
In this function you want to test if all operations of all projects are listed. However, you only create an operation in a single project.

Instead, you need to create a second project, and launch a container there as well. Then, you can run the exec commands for both containers, and count the number of operations.